internal/report/report.go

```diff
@@ -759,7 +767,15 @@ func printTraces(w io.Writer, rpt *Report) error {
 		// Print any numeric labels for the sample
 		var numLabels []string
 		for k, v := range sample.NumLabel {
-			numLabels = append(numLabels, fmt.Sprintf("%10s:  %s\n", k, strings.Trim(fmt.Sprintf("%d", v), "[]")))
+			formatTag := func(vv int64, key string) string {
+				return measurement.ScaledLabel(vv, key, o.OutputUnit)
```

Comment:
Using o.OutputUnit seems a bit weird. The output unit for the metric has nothing to do with the tag output units? There is only one output metric (set by sample_index), but there are multiple tags, so if anything I'd expect having a separate option like `-tag_output_unit=bytes:kb,alignment:bytes`. We don't need to add that flag now, but using o.OutputUnit appears strange to me.

Reply:
I agree -- updated so tag units are auto-scaled